### DISCUSSION TITLE
Error using optional config

### DISCUSSION THREAD

**Discussion by sammzinc** [linuxserver/Heimdall#991] [Q&A]
Hello
Trying to setup optional config for portainer, getting following error.
cURL error 60: SSL certificate problem: self signed certificate (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://192.168.100.144:9443/#!/home/api/auth

**Comment by Gabisonfire** [comment]
That's probably because you are using a self-signed certificate or you are accessing heimdall with it's IP instead of a valid domain name. Either change the URL to the domain name specified in the certificate or instruct portainer to use "insecure" connection. (-k in curl).

**Comment by sunbrolynk** [comment]
correct me please so i can understand, but it seems like a recent release of portainer forced HTTPS only connection, and does not allow HTTP any longer, last time i tried it doesnt launch at all on http://x.x.x.x:9000, and will only respond on https://x.x.x.x:9443 now? 
so are our only options to publish either one or the other with a domain to enable connections between them now? i am new to a lot of this but trying to understand
